docs/quickstart.md

```diff
@@ -20,6 +28,14 @@ $ kubectl config current-context
 my-cluster
 ```
 
+### Understand your Security Context
+
+As with all powerful tools, ensure you are installing it correctly for your scenario.
+
+If you're using Helm on a cluster that you completely control, like minikube or a cluster on a private network in which sharing is not a concern, the default installation -- which applies no security configuration -- is fine, and it's definitely the easiest. To install Helm without additional security steps, [install Helm](#Install-Helm) and then [initialize Helm](#initialize-helm-and-install-tiller).
+
+However, if your cluster is exposed to a larger network or if you share your cluster with others -- production clusters fall into this category -- you must take extra steps to secure your installation to prevent careless or malicious actors from damaging the cluster or its data. To apply configurations that secure Helm for use in production environments and other multitenant scenarios, see [Securing a Helm installation](securing_installation.md)
```

Comment:
consider adding a dash: "multi-tenant". Does that look good?

Reply:
ah, AP style or not? :-P I agree that adding a hyphen makes it a bit easier to parse quickly.